modules/services/lorri.nix

```diff
@@ -6,18 +6,48 @@ let
 
   cfg = config.services.lorri;
 
+  jqFile = ''
+    (
+      (.Started?   | values | "Build starting in \(.nix_file)"),
+      (.Completed? | values | "Build complete in \(.nix_file)"),
+      (.Failure?   | values | "Build failed in \(.nix_file)")
+    )
+  '';
+
+  notifyScript = pkgs.writeTextFile {
+    name = "lorri_notify";
+    executable = true;
+    text = ''
+      #! /usr/bin/env bash
+
+      lorri internal stream-events --kind live |\
+      jq --unbuffered '${jqFile}' |\
+      xargs -n 1 notify-send "Lorri Build"
+    '';
+  };
+
 in {
   meta.maintainers = [ maintainers.gerschtli ];
 
   options.services.lorri = {
     enable = mkEnableOption "lorri build daemon";
 
+    notify = mkEnableOption "lorri build notifications";
+
     package = mkOption {
       type = types.package;
       default = pkgs.lorri;
       defaultText = literalExpression "pkgs.lorri";
       description = "Which lorri package to install.";
     };
+
+    nixPackage = mkOption {
+      type = types.package;
+      default = pkgs.nix;
+      defaultText = literalExpression "pkgs.nix";
+      description = "Which nix package to use.";
+      example = literalExpression "pkgs.nixUnstable";
+    };
```

Comment:
TODO: change the default value to `config.nix.package` once we add this option.

Reply:
Not sure how to interpret this - just a note to the project, or does it need to be addressed?

Reply:
The `nix.package` option is available now but I left changing the default as an exercise for some intrepid soul 😉